src/bors/handlers/trybuild.rs

```diff
@@ -167,21 +168,36 @@ pub(super) async fn command_try_cancel<Client: RepositoryClient>(
         return Ok(());
     };
 
-    if let Err(error) = cancel_build_workflows(repo, db, &build).await {
-        tracing::error!(
-            "Could not cancel workflows for SHA {}: {error:?}",
-            build.commit_sha
-        );
-    }
-
-    db.update_build_status(&build, BuildStatus::Cancelled)
-        .await?;
-
-    tracing::info!("Try build cancelled");
+    match cancel_build_workflows(repo, db, &build).await {
+        Err(error) => {
+            tracing::error!(
+                "Could not cancel workflows for SHA {}: {error:?}",
+                build.commit_sha
+            );
+            db.update_build_status(&build, BuildStatus::Cancelled)
+                .await?;
 
-    repo.client
-        .post_comment(pr_number, "Try build cancelled.")
-        .await?;
+            repo.client
+                .post_comment(pr_number, "There was an error cancelling try build!")
```

Comment:
Let's reword this a bit, otherwise people might retry the cancelling too much. Something like "Try build was cancelled. It was not possible to cancel some workflows."